contrib/sssd.spec.in

```diff
@@ -136,6 +136,7 @@ BuildRequires: libtevent-devel
 BuildRequires: libtool
 BuildRequires: libunistring
 BuildRequires: libunistring-devel
+BuildRequires: utf8proc-devel
```

Comment:
Can we replace use of `libunistring` with `utf8proc` entirely?

Reply:
This needs `:packaging:` RN.

Reply:
> Can we replace use of `libunistring` with `utf8proc` entirely?

Maybe, we basically use `u8_check()`, `u8_tolower()` and `u8_casecmp()` from `libunistring`. For the latter two the casefolding functions might be useful. I guess for `u8_check()` we can try to identify a simple function from `uft8proc` which will return an error on an invalid UTF-8 string.

However, I would prefer to handle this separately.

Reply:
> This needs `:packaging:` RN.

added

Reply:
> we basically use `u8_check()`, `u8_tolower()` and `u8_casecmp()` from `libunistring`

and `u8_strlen()`

Reply:
> For the latter two the casefolding functions might be useful.

or plain `utf8proc_map(UTF8PROC_CASEFOLD`), without normalization..?

Reply:
#7936

Reply:
Since there is a question of availability of 'utf8proc', I'm trying an opposite approach - to consolidate on 'libunistring': #7939